pages/common/scrapy.md

```diff
@@ -0,0 +1,31 @@
+# scrapy
+
+> Web-crawling Framework.
+
+- Create a project:
+
+`scrapy startproject {{project_name}}`
+
+- Create a spider (in project directory):
+
+`scrapy genspider {{spider_name}} {{website_domain}}`
+
+- Edit spider (in project directory):
+
+`scrapy edit {{spider_name}}`
+
+- Run spider (in project directory):
+
+`scrapy crawl {{spider_name}}`
+
+- Fetch a webpage as scrapy sees it and print source in stdout:
+
+`scrapy fetch {{url}}`
+
+- View a webpage as scrapy sees it, opens it in default browser (recommended: to disable javascript beforehand):
+
+`scrapy view {{url}}`
+
+- Open scrapy shell for url, which allows interaction with the page source in ipython or python shell:
```

Comment:
Just a minor question - why was "ipython" added ? I think for a beginner level its good to just keep it to python shell.